index.html

```diff
@@ -4272,6 +4272,24 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.3.0</b> &mdash; <small><i>Aug. 4, 2015</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.1...1.3.0">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.3.0/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Collection methods <tt>find</tt>, <tt>filter</tt>, <tt>reject</tt>, <tt>every</tt>, 
+        <tt>some</tt>, and <tt>partition</tt> now support a model-attributes-style predicate 
+        <tt>this.collection.reject({user: 'guybrush'})</tt>.
+      </li>
+      <li>
+        Backbone events now support multiple-event maps <tt>obj.on({'click focus': action})</tt>.
```

Comment:
This was a bugfix. It was supported a few versions ago, but the events refactor caused a regression.